stdlib/3.4/asyncio/events.pyi

```diff
@@ -131,12 +131,12 @@ class AbstractEventLoop(metaclass=ABCMeta):
     def sock_accept(self, sock: Any) -> Any: ...
     # Signal handling.
     @abstractmethod
-    def add_signal_handler(self, sig: int, callback: Callable[[], Any], *args: List[Any]) -> None: ...
+    def add_signal_handler(self, sig: int, callback: Callable[..., Any], *args: List[Any]) -> None: ...
     @abstractmethod
     def remove_signal_handler(self, sig: int) -> None: ...
     # Error handlers.
     @abstractmethod
-    def set_exception_handler(self, handler: Callable[[], Any]) -> None: ...
+    def set_exception_handler(self, handler: Callable[..., Any]) -> None: ...
```

Comment:
This one is wrong, I'll fix that.

https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.AbstractEventLoop.set_exception_handler
> If handler is a callable object, it should have a matching signature to (loop, context), where loop will be a reference to the active event loop, context will be a dict object (see call_exception_handler() documentation for details about context).